extension/app/src/lib/asserNever.ts

```diff
@@ -0,0 +1,5 @@
+export function assertNeverAndIgnore(x: never): void {
+  console.log(
```

Comment:
We can maybe console.error?

Also maybe we can add some more comment to explain why we have to do this? Cause it's clear for us both why but maybe not everyone is aware of the contraints of the release for the Chrome extension?